api/resolvers/wallet.js

```diff
@@ -406,7 +411,7 @@ const resolvers = {
             userId: me.id,
             wallet: type ?? undefined,
             createdAt: {
-              gte: from ? new Date(Number(from)) : undefined,
+              gt: from ? new Date(Number(from)) : undefined,
```

Comment:
make sure we don't fetch the last log over and over again. maybe not significant in terms of bandwidth but I think non-overlapping pages are better.